browser/containers/Report.tsx

```diff
@@ -0,0 +1,133 @@
+/* Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+import * as React from 'react';
+import { CSVLink } from 'react-csv';
+import ReactTable from 'react-table';
+import { reqJSON } from '../util/index';
+
+interface Props {
+  params: any;
+}
+
+interface State {
+  title: string;
+  description: string;
+  tables: Array<{
+    title?: string;
+    columns: Array<{
+      name: string;
+      db: string;
+    }>;
+    data: any[]; // this changes depending on the report
```

Comment:
You should define all available options and mark the non-universal ones as optional. See this, https://www.typescriptlang.org/docs/handbook/interfaces.html, for more information.